narwhals/_dask/dataframe.py

```diff
@@ -136,6 +140,11 @@ def with_row_index(self: Self, name: str) -> Self:
     def rename(self: Self, mapping: dict[str, str]) -> Self:
         return self._from_native_dataframe(self._native_dataframe.rename(columns=mapping))
 
+    def head(self: Self, n: int) -> Self:
+        return self._from_native_dataframe(
+            self._native_dataframe.head(n=n, compute=False)
```

Comment:
Should we also pass `npartititions=-1` to parse all partitions if needed? I am not sure how this works: [docs](https://docs.dask.org/en/latest/generated/dask.dataframe.DataFrame.head.html)

Reply:
good points, thanks